pages/documentation/developer/tutorials/sensitivity-analysis.md

```diff
@@ -239,17 +246,10 @@ In this provider, we first define the variable of interest: here the branch flow
 
 Now the sensitivity inputs are prepared, we can run a sensitivity analysis. This is done in the following way:
 ```java
-ComputationManager computationManager = LocalComputationManager.getDefault();
-SensitivityComputation sensitivityComputation = new Hades2SensitivityFactory().create(network,
-    computationManager, 1);
-SensitivityComputationResults sensiResults = sensitivityComputation.run(factorsProvider,
-    VariantManagerConstants.INITIAL_VARIANT_ID, SensitivityComputationParameters.load()).join();
-
+SensitivityAnalysisResults sensiResults = SensitivityAnalysis.find().run(network, factorsProvider);
 ```
-In order to run the calculation, we need to create a `ComputationManager` (<span style="color: red">TODO: explain its role</span>), 
-and a `SensitivityComputation` object based on the Hades2 implementation.
-Then we can run the calcuation itself, using the initial netork variant as network data.
-Here we directly loaded the sensitivity analysis parameters from the YML configuration file in the resources.
+When no variants are explicitely specified, the analysis will be performed on network working variant.
+Here we directly load the sensitivity analysis parameters from the YML configuration file in the resources.
```

Comment:
From the resources or the configuration directory?

Reply:
Don't take into account this comment: the tutorial is looking for the parameters in the resources :)